be/src/olap/data_dir.cpp

```diff
@@ -486,6 +505,13 @@ Status DataDir::load() {
             continue;
         }
 
+        if (rowset_meta->partition_id() == 0) {
+            LOG(WARNING) << "skip tablet_id=" << tablet->tablet_id()
```

Comment:
If there is a committed rowset with partition id = 0; and you ignore it. Then publish version will success and FE will finish the txn. But BE will not have this data, then data loss and query will failed